src/collections/resources/articles/kubernetes/simplifying-kubernetes-security/index.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "Simplifying Kubernetes Security"
+date: 2022-12-01 10:30:05 -0530
+author: Tolulope Ola-David
+thumbnail: ./kubernetes-logo.svg
+darkthumbnail: ./kubernetes-logo.svg
+description: Simplifying Kubernetes Security
+type: Article
+published: true
+category: Kubernetes
+tags:
+ - Kubernetes
+resource: true
+---
+import { ResourcesWrapper } from "../../../Resources.style.js";
+import ClusterImg from "./multi-cluster-kubernetes-management-with-meshery.png";
+
+<ResourcesWrapper>
+
+  <h3> Introduction </h3>
+
+  <p> As businesses become increasingly reliant on cloud-based technologies, securing cloud workloads is becoming 
+  more important than ever. Kubernetes has emerged as one of the most popular solutions for managing 
+  cloud-based workloads. But how does Kubernetes' security work? </p>
+
+  <p> Kubernetes security is an important topic for businesses and organizations that rely on Kubernetes for 
+  their cloud native applications. Kubernetes is quickly becoming the standard for container orchestration 
+  and deployment, and it's important to understand the security implications of using this technology. This 
+  article explains the basics of Kubernetes security, from authentication and authorisation to network 
+  security and more. It also looks at the various tools available to help you secure your Kubernetes 
+  clusters and the best practices for keeping your environment safe. </p>
+
+  <p> The article also provides an overview of the impact of microservices and the best practices for 
```

Comment:
This seems a little repetitive. This para can be completely skipped, or maybe merged into the last paragraph. @deepeshaburse what do you think?

Reply:
@eeshaanSA I agree, thanks for pointing it out. I have removed that part and added it to the previous paragraph. Does it look good to you now? And, we don't have images for this topic, unfortunately.